scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala

```diff
@@ -54,6 +54,12 @@ class KryoHadoop(config: Config) extends KryoInstantiator {
    */
   override def newKryo: Kryo = {
     val newK = (new ScalaKryoInstantiator).newKryo
+    if (!newK.alreadyRegistered[Stream[Any]]) {
```

Comment:
Should we open some issue to kill this again, lest we forget... Though we'll probably forget about the issue too